### PR TITLE
Nama

### PR DESCRIPTION
**Pull request by theunknownSAI**
Please review the assignment @rehana-ms ma'am

### CODE REVIEW

File: project1/templates/search.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+              class="btn btn-success"
+              type="submit"
+              formaction="{{url_for('search')}}"
+            >
+              search
+            </button>
+          </form>
+        </div>
+        <div class="card col-3" style="width: 18rem;">
+          <ul class="list-group list-group-flush">
```
Comment (rehana-ms) [inline project1/templates/search.html:63]: Can you tell me what is the purpose of this list?

File: project1/application.py [hunk 2]
```diff
@@ -101,15 +105,28 @@ def auth():
...
 
 
 @app.route("/search", methods=["GET", "POST"])
 def search():
-    return " search "
+    if request.method == "POST":
+        details = request.form.get('search')
+        details = "%"+details+"%"
+        books = Book.query.filter(
+            or_(Book.isbn.like(details), Book.title.like(
```
Comment (rehana-ms) [inline project1/application.py:116]: are you able to fetch partial results also.Can you give me an example for your search query.

File: project1/templates/search.html (hunk 1 above)
Comment (theunknownSAI) [reply project1/templates/search.html:63]: This shows the format of search results.

File: project1/application.py (hunk 2 above)
Comment (theunknownSAI) [reply project1/application.py:116]: Yes,
if details = 256
 details = "%"+details+"%" ... this says anything consisting of 256.
so in the query, it results all the match details.

Comment (rehana-ms) [reply project1/application.py:116]: Can you give lion as search query and share the screenshot pls.

File: project1/templates/search.html (hunk 1 above)
Comment (rehana-ms) [reply project1/templates/search.html:63]: I suggest to ignore this.

File: project1/application.py (hunk 2 above)
Comment (theunknownSAI) [reply project1/application.py:116]: search

**lion** as input

Comment (rehana-ms) [reply project1/application.py:116]: good.One suggestion is  spacing between each book can be reduced.Then it looks very nice.

Comment (theunknownSAI) [reply project1/application.py:116]: OK Ma'am

Comment (rehana-ms) [reply project1/application.py:116]: Just now I recognized that Dandelion wine appeared twice in your search result.Can you fix that  bug.

Comment (theunknownSAI) [reply project1/application.py:116]: Done Ma'am

Comment (rehana-ms) [reply project1/application.py:116]: can you share the result for lion now.

Comment (theunknownSAI) [reply project1/application.py:116]: Capture

Comment (rehana-ms) [reply project1/application.py:116]: good

**Pull request merged**